sdk/communication/AzureCommunication/Source/Generated/Enumerations.swift

```diff
@@ -0,0 +1,63 @@
+// --------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// --------------------------------------------------------------------------
+
+import AzureCore
+import Foundation
+
+/// The cloud that the identifier belongs to.
+public enum CommunicationCloudEnvironmentModel: RequestStringConvertible, Codable, Equatable {
+    /// Custom value for unrecognized enum values
+    case custom(String)
+
+    case `public`
```

Comment:
I have some concerns here. Because `public` is a keyword this case needs to be put inside the ` `` `. 
The next time we generate this file the compiler should throw us an error at compile time. So should be easy enough to fix.

Reply:
I didn't realize that any part of AzureCommunication was generated?  However, please file an issue in the autorest.swift repo for this.

Reply:
I will file an issue for this

Reply:
https://github.com/Azure/autorest.swift/issues/299